migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc

```diff
@@ -46,3 +47,14 @@ You can roll back a migration by using the {mtc-short} web console or the CLI.
 
```

Comment:
Should this assembly also now include a line about being able to roll back manually? (and possibly a link, as in my comment on the other assembly)